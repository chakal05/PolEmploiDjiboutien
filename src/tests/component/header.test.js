import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../components/header';

test('should render header component', () => {
	const renderer = new ShallowRenderer();
    renderer.render(<Header />);
   expect(renderer.getRenderOutput()).toMatchSnapshot();
});
