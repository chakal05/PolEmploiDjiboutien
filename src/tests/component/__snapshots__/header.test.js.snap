// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header component 1`] = `
<ForwardRef(WithStyles)
  className="AppBar"
  elevation={0}
  position="static"
>
  <ForwardRef(WithStyles)>
    <a
      className="link"
      href="/"
    >
       
      Kerja
       
    </a>
  </ForwardRef(WithStyles)>
</ForwardRef(WithStyles)>
`;
